pkg/clients/database/mysql.go

```diff
@@ -77,18 +74,27 @@ type MySQLServerClient struct {
 func NewMySQLServerClient(c *azure.Client) (*MySQLServerClient, error) {
 	mysqlServersClient := mysql.NewServersClient(c.SubscriptionID)
 	mysqlServersClient.Authorizer = c.Authorizer
-	mysqlServersClient.AddToUserAgent(azure.UserAgent)
+	if err := mysqlServersClient.AddToUserAgent(azure.UserAgent); err != nil {
+		return nil, err
+	}
```

Comment:
That's the function:
```
if extension != "" {
	c.UserAgent = fmt.Sprintf("%s %s", c.UserAgent, extension)
	return nil
}
return fmt.Errorf("Extension was empty, User Agent stayed as %s", c.UserAgent)
```

And we put a string constant there meaning it's pretty hard to make it return error in the way we call it. But the function says that it can, so, I just wanted to respect it. I thought they might change the underlying implementation and feel safe since the signature didn't change and we get errors on runtime.